.gitignore

```diff
@@ -1,2 +1,104 @@
 *.class
 *.log
+
+# sbt specific
+dist/*
+target/
```

Comment:
`**/target` - for targets in 'project', 'project/project' etc.

Reply:
means? this sbt config generate multi project target builds (core/targets etc), and it is logically exclude all the builds

Reply:
`target/` includes all appearances of target folder?